services/movie_session.py

```diff
@@ -0,0 +1,49 @@
+from django.utils.datetime_safe import datetime
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id,
+    )
+    return movie_session
+
+
+def get_movies_sessions(session_date: datetime.date = None) -> MovieSession:
```

Comment:
The return type annotation for `get_movies_sessions` should be `QuerySet[MovieSession]` instead of `MovieSession`, as it returns a queryset of movie sessions.